container.go

```diff
@@ -351,6 +301,38 @@ func createContainer(pod *Pod, contConfig ContainerConfig) (*Container, error) {
 		c.mounts = mounts
 	}
 
+	return c, nil
+}
+
+func createContainers(pod *Pod, contConfigs []ContainerConfig) ([]*Container, error) {
```

Comment:
why empty containers ? it calls into newContainer and it creates Container internal structures based on containerConfig, right ?